.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description of changes:
+
+## CTSM Issues Fixed (include github issue #):
+(https://github.com/ESCOMP/ctsm/issues)
+
+## Are answers expected to change?:
+
+## Any User Interface Changes?:
```

Comment:
What do you mean by this? Does this mean "Any new xml or namelist variables"? If so, we should say that explicitly rather than making people guess what we mean by user interface changes.